ds18b20/doc.go

```diff
@@ -21,6 +21,7 @@
 // Datasheets
 //
 // https://datasheets.maximintegrated.com/en/ds/DS18B20-PAR.pdf
+// https://datasheets.maximintegrated.com/en/ds/DS18S20.pdf
```

Comment:
Technically it's better to leave an empty line so it shows up better at https://periph.io/x/devices/v3/ds18b20. I realize the other link should be changed to https. The two can be done in a follow up.